src/bloqade/task/braket.py

```diff
@@ -30,30 +40,78 @@ class BraketTask(RemoteTask):
     )
 
     def submit(self, force: bool = False) -> "BraketTask":
+        """
+        Submits the task to the backend.
+
+        Args:
+            force (bool): Whether to force submission even if the task is already submitted.
+
+        Returns:
+            BraketTask: The current task instance.
+
+        Raises:
+            ValueError: If the task is already submitted and force is False.
+
+        Example:
+            >>> backend = BraketBackend(...)  # Create a backend instance
```

Comment:
This won't be rendered correctly by mkdocs markdown. You need to wrap them with markdown code blocks.